go/libkb/sig_chain.go

```diff
@@ -48,6 +48,11 @@ type SigChain struct {
 	idVerified        bool
 	loadedFromLinkOne bool
```

Comment:
`loadedFromLinkOne` seems like dead code. But I'm not sure how I would be calculating hPrevs if this was ever set `true`. Should I remove it?